# Parameters for the module names, gains, hrdf, etc
names: [ "Base", "Shoulder", "Elbow", "Wrist1" ]
families: [ "Arm" ]
gains_package: "hebi_cpp_api_examples"
gains_file: "config/gains/a-2085-04_gains.xml"
hrdf_package: "hebi_cpp_api_examples"
hrdf_file: "config/hrdf/a-2085-04.hrdf"
home_position: [ 0.01, 2.09439, 2.09439, 0.01 ]

# Note: the "moveit_joints" must matches the configuration from the MoveIt config; only
# required when running MoveIt examples
moveit_joints: ["Arm/Base/X5_4", "Arm/Shoulder/X5_9", "Arm/Elbow/X5_4", "Arm/Wrist1/X5_1"]
